test/Mono.Linker.Tests.Cases/LinkAttributes/OverrideAttributeRemoval.xml

```diff
@@ -3,4 +3,7 @@
   <assembly fullname="Mono.Linker.Tests.Cases.Expectations">
```

Comment:
The one from test assembly should not be needed anymore.

Reply:
It'll be required for mono test run